src/misc/programs.cpp

```diff
@@ -891,76 +878,129 @@ void PROGRAMS_Destroy([[maybe_unused]] Section* sec)
+	MSG_Add("PROGRAM_CONFIG_FILE_WHICH", "Writing config file '%s'\n");
 
-	// help
+	// Help
 	MSG_Add("SHELL_CMD_CONFIG_HELP_LONG",
-	        "Adjusts DOSBox Staging's configurable parameters.\n"
-	        "-writeconf or -wc without parameter: write to primary loaded config file.\n"
-	        "-writeconf or -wc with filename: write file to config directory.\n"
-	        "Use -writelang or -wl filename to write the current language strings.\n"
-	        "-r [parameters]\n"
-	        " Restart DOSBox, either using the previous parameters or any that are appended.\n"
-	        "-wcp [filename]\n"
-	        " Write config file to the program directory, dosbox.conf or the specified\n"
-	        " filename.\n"
-	        "-wcd\n"
-	        " Write to the default config file in the config directory.\n"
-	        "-l lists configuration parameters.\n"
-	        "-h, -help, -? sections / sectionname / propertyname\n"
-	        " Without parameters, displays this help screen. Add \"sections\" for a list of\n"
-	        " sections."
-	        " For info about a specific section or property add its name behind.\n"
-	        "-axclear clears the autoexec section.\n"
-	        "-axadd [line] adds a line to the autoexec section.\n"
-	        "-axtype prints the content of the autoexec section.\n"
-	        "-securemode switches to secure mode.\n"
-	        "-avistart starts AVI recording.\n"
-	        "-avistop stops AVI recording.\n"
-	        "-startmapper starts the keymapper.\n"
-	        "-get \"section property\" returns the value of the property.\n"
-	        "-set \"section property=value\" sets the value.\n");
+	        "Performs configuration management tasks and other miscellaneous actions.\n"
+	        "\n"
+	        "Usage:\n"
+	        "  [color=green]config[reset] [color=white]COMMAND[/reset] [color=cyan][PARAMETERS][/reset]\n"
+	        "\n"
+	        "Where [color=white]COMMAND[/reset] is one of:\n"
+	        "  -writeconf\n"
+	        "  -wc               Writes the config to the primary config file.\n"
+	        "\n"
+	        "  -writeconf [color=white]FILENAME[/reset]\n"
+	        "  -wc [color=white]PATH          [/reset]Writes the config to [color=white]PATH[/reset] relative to the config directory,\n"
+	        "                    or to an absolute path.\n"
+	        "\n"
+	        "  -wcp [color=white]FILENAME     [/reset]Writes the config to the current working directory as\n"
+	        "                    'dosbox.conf', or as the specified filename.\n"
+	        "  -wcd              Writes the config to the default global config file.\n"
+	        "\n"
+	        "  -writelang [color=white]FILENAME[/reset]\n"
+	        "  -wl [color=white]FILENAME      [/reset]Writes the current language strings to [color=white]FILENAME [/reset]in the\n"
+	        "                    current working directory.\n"
+	        "\n"
+	        "  -r [color=cyan][PROPERTY1=VALUE1][ PROPERTY2=VALUE2]...[/reset]\n"
```

Comment:
`[ PROPERTY2=VALUE2]` - space is not needed here. I suggest:
`[PROPERTY1=VALUE1][ PROPERTY2=VALUE2]...` -> `[PROPERTY1=VALUE1 [PROPERTY2=VALUE2 ...]]`

Reply:
Sure, that's better.